twoliter/src/lock.rs

```diff
@@ -139,6 +170,111 @@ impl Lock {
         Ok(lock)
     }
 
+    /// Fetches all external kits defined in a Twoliter.lock to the build directory
+    pub(crate) async fn fetch(&self, project: &Project, arch: &str) -> Result<()> {
+        let target_dir = project.project_dir().join("build/external-kits");
```

Comment:
Should this be a `CONSTANT` or else have some accessor like `project.external_kits_dir()`?